hashicorp/waypoint
# ui: upgrade ember-auto-import to ^2.2.4

## Why the change?

Just keeping things up-to-date (and because we can now that we’re on Node 14)

## How does it affect build times?

Comparing the ember-build-tests time between `main` (1m17s) and this branch (1m25s) doesn’t show any significant difference.

## What does it look like?

Totally normal, hopefully.

<img width="1316" alt="CleanShot 2021-11-17 at 12 48 18@2x" src="https://user-images.githubusercontent.com/34030/142195191-c2ba2744-c072-467e-9de7-df30222f8c0c.png">

## How do I test it?

1. Check out the branch
2. Verify dev mode works as you’d expect
3. Verify it works when compiled into the server binary

## Anything else to consider?

Yeah, with this change a bunch of stuff moves out of `vendor.js` and into several `chunk.*.js` files. I’m not yet sure if our ember-asset-size action will account for this.
